File: lib/src/model/flutter_call_info.dart
import 'package:flutter_bridge/src/model/flutter_method_info.dart';

///   author : liuduo
///   e-mail : liuduo@gyenno.com
///   time   : 2022/08/27
///   desc   : call信息
///   version: 1.0
class FlutterCallInfo {
  ///method信息
  FlutterMethodInfo methodInfo;

  ///数据
  dynamic data;

  FlutterCallInfo(this.methodInfo, this.data);

  @override
  String toString() {
    return 'FlutterCallInfo{methodInfo: $methodInfo, data: $data}';
  }
}

File: android/src/main/java/com/dodo/flutterbridge/model/FlutterCallInfo.kt
package com.dodo.flutterbridge.model

/**
 *     author : liuduo
 *     e-mail : liuduo@gyenno.com
 *     time   : 2022/08/12
 *     desc   : call信息
 *     version: 1.0
 */
data class FlutterCallInfo(
    /**
     * method信息
     */
    val methodInfo: FlutterMethodInfo,
    /**
     * 数据
     */
    val data: Any?
)